apps/docs/app/ui/components/previewComponent/CodeExample.tsx

```diff
@@ -0,0 +1,17 @@
+import { getFileContent } from "@/app/lib/getFileContent.ts";
+import { CodeBlock } from "@/app/ui/components/codeBlock/CodeBlock.tsx";
+import clsx from "clsx";
+
+const CodeExample = async ({ name, key, className }: { name: string; key: string; className?: string }) => {
```

Comment:
This comment will be repeated a couple of time: 
- Create an interface instead of inline types. 
- is the key prop really necessary?